README.md

```diff
@@ -133,10 +132,48 @@ us a report nonetheless.
 
 When in doubt, please do send us a report.
 
-
 ## Current Project Team Members
 
-_To be written_
+The Ayo.js project team is comprised of a core team, a moderation team, and
+various sub-teams. For more information, see [GOVERNANCE.md](./GOVERNANCE.md).
+
+### Core Team
+
+- [addaleax](https://github.com/addaleax) -
+**Anna Henningsen** &lt;anna@addaleax.net&gt; (she/her)
+- [aqrln](https://github.com/aqrln) -
+**Alexey Orlenko** &lt;eaglexrlnk@gmail.com&gt; (he/him)
+- [Harrison-M](https://github.com/Harrison-M) -
+**Harrison Massey** &lt;harrison.massey@gmail.com&gt;
+- [pup](https://github.com/pup) -
+**Olivia Hugger** &lt;olivia@fastmail.com&gt; (she/her)
+- [Qard](https://github.com/Qard) -
+**Stephen Belanger** &lt;admin@stephenbelanger.com&gt;
+- [sandfox](https://github.com/sandfox) -
+**James Butler** &lt;james.butler@sandfox.co.uk&gt;
+- [TimothyGu](https://github.com/TimothyGu] -
```

Comment:
Syntax seems to be broken. 
<img width="767" alt="2017-09-19 19 37 25" src="https://user-images.githubusercontent.com/3367801/30588553-16d9ef72-9d72-11e7-9e52-0027df3c4c6e.png">

```diff
- - [TimothyGu](https://github.com/TimothyGu] -
+ - [TimothyGu](https://github.com/TimothyGu) -
```

Reply:
whoops

Reply:
fixed